<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello Notepad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_saveas"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>편집</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
    <addaction name="separator"/>
    <addaction name="action_15"/>
    <addaction name="action_f5"/>
   </widget>
   <widget class="QMenu" name="menu_form">
    <property name="title">
     <string>서식</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>보기</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>도움말</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_form"/>
   <addaction name="menu_view"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_open">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>열기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_saveas">
   <property name="text">
    <string>다른 이름으로 저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>끝내기</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>실행취소</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>잘라내기</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>복사</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>붙여넣기</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>찾기</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>다음 찾기</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>이전 찾기</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>바꾸기 </string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>이동 </string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>모두 선택</string>
   </property>
  </action>
  <action name="action_f5">
   <property name="text">
    <string>시간 / 날짜 f5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
